staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -185,6 +205,13 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 	case "sign", "asInteger", "isInteger", "asApproximateFloat", "isGreaterThan", "isLessThan", "compareTo", "add", "sub":
 		cost := uint64(1)
 		return &cost
+	case "getScheme", "getHostname", "getHost", "getPort", "getEscapedPath", "getQuery":
```

Comment:
this is good example of unhandled paths which got missed and were hitting the default cost of 1 ... fortunately that is a reasonable cost for these, but we could easily have missed cases for expensive things

Reply:
+100